nautobot/docs/release-notes/version-2.2.md

```diff
@@ -12,16 +12,28 @@ This document describes all new features and changes in Nautobot 2.2.
 
 Contact and Team are models that represent an individual and a group of individuals who can be linked to an object. Contacts and teams store the necessary information (name, phone number, email, and address) to uniquely identify and contact them. They are added to track ownerships of organizational entities and to manage resources more efficiently in Nautobot. Check out the documentation for [contact](../user-guide/core-data-model/extras/contact.md) and [team](../user-guide/core-data-model/extras/team.md). There is also a [user guide](../user-guide/feature-guides/contact-and-team.md) available on how to utilize these models.
 
-#### Software Image File and Software Version models ([#1](https://github.com/nautobot/nautobot/issues/1))
+#### Controller Model ([#3111](https://github.com/nautobot/nautobot/issues/3111))
 
-New models have been added for software image files and software versions. These models are used to track the software versions of devices, inventory items and virtual machines and their associated image files. These models have been ported from the [Device Lifecycle Management App](https://github.com/nautobot/nautobot-app-device-lifecycle-mgmt/) and a future update to that app will migrate all existing data from the `nautobot_device_lifecycle_mgmt.SoftwareImageLCM` and `nautobot_device_lifecycle_mgmt.SoftwareLCM` models to the `dcim.SoftwareImageFile` and `dcim.SoftwareVersion` models added here.
+A Controller in Nautobot is an abstraction meant to represent network or SDN (Software-Defined Networking) controllers. These may include, but are not limited to, wireless controllers, cloud-based network management systems, and other forms of central network control mechanisms. See more details in the model [documentation](../user-guide/core-data-model/dcim/controller.md) and see the remaining work outlined in this GitHub [issue](https://github.com/nautobot/nautobot/issues/5454).
```

Comment:
I'd leave out the reference to #5454, both because it should be resolved before the final release and because we don't generally call out things not yet done in the release overview.

```suggestion
A Controller in Nautobot is an abstraction meant to represent network or SDN (Software-Defined Networking) controllers. These may include, but are not limited to, wireless controllers, cloud-based network management systems, and other forms of central network control mechanisms. See more details in the model [documentation](../user-guide/core-data-model/dcim/controller.md).
```